addon/index.js

```diff
@@ -201,7 +202,7 @@ export function changeset(obj, validateFn = defaultValidatorFn, validationMap =
 
         setProperties(content, changes);
 
-        this._allKeys(resetProperties).forEach(function(key) {
+        recursiveKeys(resetProperties).forEach(function(key) {
           if (!changedKeys.includes(key)) {
```

Comment:
I missed this, but let's wrap `changedKeys` with `Ember.A` here to ensure the `.includes` method exists.

Reply:
(Unsure what the new Ember import would be here.)